skate-plugin/src/main/kotlin/com/slack/sgp/intellij/SqlDelightErrorHandler.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2016 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.sgp.intellij
+
+import com.bugsnag.Bugsnag
+import com.bugsnag.Severity
+import com.intellij.diagnostic.AbstractMessage
+import com.intellij.ide.plugins.PluginManagerCore
+import com.intellij.openapi.application.ApplicationInfo
+import com.intellij.openapi.diagnostic.ErrorReportSubmitter
+import com.intellij.openapi.diagnostic.IdeaLoggingEvent
+import com.intellij.openapi.diagnostic.SubmittedReportInfo
+import com.intellij.util.Consumer
+import java.awt.Component
+
+/**
+ * Using SqlDelightErrorHandler to use for crash reporting for the Skate plugin. Currently just have
+ * placeholders for GIT_SHA, VERSION, and BUGSNAG_KEY
+ */
+class SqlDelightErrorHandler : ErrorReportSubmitter() {
```

Comment:
Please update the names for everything to align with the skate plugin, not sqldelight